templates/app/angular-default/template/.eslintrc.json

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
I assume eslint can handle json with comments like this file? Any reason why not just stick with JS extension so comments are "officially" supported?

Reply:
I wanted to stick with the behavior in a web based Angular project where the [angular-eslint](https://github.com/angular-eslint/angular-eslint#step-2---add-new-eslint-related-configuration) command also adds this exact file.